app.py

```diff
@@ -16,6 +17,25 @@
 
 app = Flask(__name__)
 
+#Enter your database URL here , Get from Heroku Database Crdentials
+
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS']=True
+os.environ['DATABASE_URL'] = 'postgres://nqzbhrnyrmcwsp:371a5e4edb0270cdfc1e494c91178274bf17edf4b4d0cc4e5a0e20a9080262e9@ec2-54-225-119-223.compute-1.amazonaws.com:5432/dft9ctgao0i98a'
```

Comment:
Don't type it here. The point of using env variables is that stuff like this is hidden in the code.